packages/asset-swapper/test/quote_report_generator_test.ts

```diff
@@ -58,32 +58,18 @@ describe('generateQuoteReport', async () => {
     it('should generate report properly for sell', () => {
         const marketOperation: MarketOperation = MarketOperation.Sell;
 
-        const kyberSample1: DexSample = {
-            source: ERC20BridgeSource.Kyber,
-            input: new BigNumber(10000),
-            output: new BigNumber(10001),
-            fillData: {},
-        };
         const kyberSample2: DexSample = {
```

Comment:
super nit: now that `kyberSample1` is removed, can this just be named `kyberSample`. Same for uni below